LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -1317,12 +1317,28 @@ function viewHistory(){
 
     $.ajax({
         type: 'GET',
-        url: 'ajaxIndex.php?a=gethistory&type=workflow&id='+currentWorkflow,
+        url: 'ajaxIndex.php?a=gethistoryText',
         dataType: 'text',
         success: function(res) {
             dialog_simple.setContent(res);
             dialog_simple.indicateIdle();
             dialog_simple.show();
+
+            setTimeout(function () {
```

Comment:
Why is this timeout necessary?

Reply:
It requires the initial response to have loaded the page content that has the "id" we'll be referencing in the function called within the timeout.